roles/rsyslog/tasks/deploy.yml

```diff
@@ -1,5 +1,5 @@
 ---
-# SPDX-License-Identifier: GPLv3
+# SPDX-License-Identifier: GPL-3.0
```

Comment:
Is this a valid SPDX identifier? I don't see it in the list.

Reply:
Yikes. You are right... It is in the deprecated list. :(

Reply:
Sorry, it is there, but among the deprecated ones.